langchain/chains/query_constructor/base.py

```diff
@@ -70,15 +73,22 @@ def _get_prompt(
     examples: Optional[List] = None,
     allowed_comparators: Optional[Sequence[Comparator]] = None,
     allowed_operators: Optional[Sequence[Operator]] = None,
+    include_k: bool = False,
```

Comment:
`enable_limit` or something like that?

`k` is will become difficult to understand as this evolves